### ISSUE TITLE
Phase banner placement guidance incorrect and shares same background as cookie banner

### ISSUE THREAD

**Issue by jrbarnes9** [ONSdigital/design-system#2325]
Phase banner has no landmark role so it should be placed within `<header>` to be contained within the same, single "banner" landmark. This is the case when using base page template, but if not, the guidance only states:

> The phase banner must be placed directly above the ONS [header](https://ons-design-system.netlify.app/components/header) at the very top of the page.

...so it's not clear that it should be within `{% block header %}`

It also shares the same background as cookie banner so there is no distinction between the two banners.

![Screenshot 2022-07-14 at 15 09 29](https://user-images.githubusercontent.com/43346934/179002631-9d079fda-4be5-4777-803e-4fc83b894cfe.png)

_Originally posted by @jrbarnes9 in https://github.com/ONSdigital/design-system/discussions/8#discussioncomment-3147512_

**Comment by yatesn** [comment]
Create ticket for documentation - Oscar

**Comment by alessioventuriniAND** [comment]
Do we need to update the colour of the phase banner? @emilymaylawes can you please advice on this? 

I have checked the documentations and it's now clear where to place the phase banner within the layout . Below if the doc reference : 

Base page template
The phase banner component can be added to the top of the header using the [base page template](https://service-manual.ons.gov.uk/design-system/foundations/base-page-template).

You can override this by manually adding it using the header block.

**Comment by emilymaylawes** [comment]
Design pushed into change looks like the below

![Image](https://github.com/ONSdigital/design-system/assets/131370908/bafa5e60-a3d3-45a6-b018-03b9126e8079)

Replacing the phase banner background colour with white, and adding a 1px bottom border with `var(--ons-color-borders)` to distinguish from both the cookie banner and the top section of the header that holds the ONS logo.
